cmd/README.md

```diff
@@ -0,0 +1,33 @@
+## Key Sign Tool
+
+Rosetta CLI has a key sign tool, which you can use to sign and verify various curves supported
+by rosetta-specifications. This should only be used for local development. Never share private keys anywhere.
+
+### Usage
+#### Sign
+```
+rosetta-cli key:sign --configuration-file config.json
+```
+
+A sample config file is located [here](../examples/configuration/sign.json)
+
+Required fields includes
+- `pub_key`
+- `private_key`
+- `signing_payload`
+
+
+#### Verify
+```
+rosetta-cli key:verify --configuration-file config.json
+```
+A sample config file is located [here](../examples/configuration/sign.json)
```

Comment:
please separate the examples to verify.json and sign.json

Reply:
Done